### PR TITLE
 About Page

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review the code for my Ecommerce Site -Booklet's About Us Page

The page includes:

- An Announcement Bar of a 10% Discount on your First Order.
- A Header with includes Logo, Search Bar, Sign Up Button, Login Button, Wishlist Button, Cart Button.
- A Nav Bar which includes Categories
- You can see Sample About Us details
- A Footer with all my social media and other Booklet page links

https://user-images.githubusercontent.com/65448358/160010179-5261963d-0500-42c8-9212-58d231b3f721.mp4

### CODE REVIEW

File: about/about.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Parcel Sandbox</title>
+		<meta charset="UTF-8" />
+		<link rel="stylesheet" href="../src/styles.css" />
```
Comment (bharati-21) [inline about/about.html]: Suggestion - You can add meta author and desc tags

**Review by bharati-21 — APPROVED** [review]
Looks good to go. You could add the meta tags. They would help you with website accessibility

**Review by k-rahul21 — APPROVED** [review]

**Comment by k-rahul21** [comment]
use meta tag for website name and for description also.

**Comment by khushijohri2001** [comment]
Done with the changes @k-rahul21 @bharati-21 thanks for reviewing.

**Pull request merged**